lib/background.js

```diff
@@ -1,45 +1,47 @@
-function Background(options) {
-  this.starfield = options.starfield;
-  this.starfieldCtx = options.starfieldCtx;
-  this.canvas = options.canvas;
-}
+class Background {
+  constructor(options) {
+    this.starfield = options.starfield;
+    this.starfieldCtx = options.starfieldCtx;
+    this.canvas = options.canvas;
+  }
 
-Background.prototype.randomStarsImage = function(){
-  this.getStars();
+  randomStarsImage() {
+    this.getStars();
 
-  var img = document.createElement("img");
-  img.src = this.starfield.toDataURL();
-  return img;
-};
+    var img = document.createElement("img");
```

Comment:
I suggest using `let` and avoiding the use of `var` due to unexpected behavior of `var`